src/components/earlyadopter/ConnectAccount.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+  <div class="pt-20 pb-20 status-background">
+    <div class="max-w-6xl mx-auto">
+      <div class="justify-center">
+        <h1 class="font-bold text-xl">
+          Early adopter participation confirmation
+        </h1>
+        <div>
+          <div>
+            <div
+              class="
+                mt-1
+                w-full
+                rounded-md
+                bg-white
+                dark:bg-darkGray-900
+                border border-gray-300
+                dark:border-darkGray-500
+              "
+            >
+              <div
+                v-if="!showConnectMetamsk"
+                class="transform text-gray-400 dark:text-darkGray-500"
+              >
+                Loading....
+              </div>
+
+              <h3
+                v-if="showConnectMetamsk"
+                class="
+                  text-lg
+                  font-extrabold
+                  text-blue-900
+                  dark:text-white
+                  mb-6
+                  text-center
+                "
+              >
+                Choose Account that you want to apply the lockdrop bonus.
+              </h3>
+              <ul
+                class="
+                  max-h-56
+                  rounded-md
+                  py-1
+                  text-base
+                  overflow-auto
+                  focus:outline-none
+                "
+              >
+                <AccountOption
+                  v-for="(account, index) in allAccounts"
+                  :key="index"
+                  :key-idx="index"
+                  :address="account"
+                  :addressName="allAccountNames[index]"
+                  :contributed="allContributedD[index]"
+                  :checked="selAccount === index"
+                  v-model:selOption="selAccount"
+                />
+              </ul>
+
+              <div class="border-2 border-dashed" />
+
+              <div class="my-2">
+                <MetamaskOption
+                  v-show="showConnectMetamsk"
+                  @connectMetamask="connectMetamask"
+                />
+                <div v-if="isResultReward" class="mt-3 font-bold">
+                  <div v-if="prevLockdropInfo">
+                    You deserve a bonus:
```

Comment:
The text should say,
`This account is eligible for a early bird bonus`

And the other text should say,
`This account did not participate in the Lockdrop` (we assume that all the polkadot accounts displayed on the page have participated in the polkadot crowdloan).